### PR TITLE
Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by kambereBr**
In this pull request I:
- Added the middle section of the portfolio that contains: Works section and About Myself section
- Implemented website responsiveness for desktop

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -119,6 +122,210 @@ header {
...
+}
+
+.works-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  row-gap: 100px;
+  background-color: #fbf8f8;
+  margin-bottom: 40px;
+}
```
Comment (Efiamotu-1) [inline assets/css/style.css:192]: - [ ] From the project requirement this `.works-section` which serves as the container for the whole works card should be styled using the css grid layour while each card should use the display flex. please kindly adhere to the project requirement as shown below and make the required changes :+1: 
- project requirements
req

File: assets/css/style.css [hunk 2]
```diff
@@ -119,6 +122,210 @@ header {
...
+
+.div-works {
+  /* display: flex;
+  flex-direction: column; */
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+
+  /* border: 3px solid; */
```
Comment (Efiamotu-1) [inline assets/css/style.css:166]: - please i will kindly suggest you remove all commented code from your code base as it is unprofessional to leave commented code.This is not in line with clean code practice :+1:

File: index.html [hunk 3]
```diff
@@ -40,5 +44,191 @@ <h4>LET’S CONNECT</h4>
...
+                <ul class="type_dev">
+                    <li class="li-color">CANOPY</li>
+                    <li><img src="assets/images/Counter.svg" alt="Counter image"></li>
+                    <li>Back End Dev</li>
+                    <li><img src="assets/images/Counter.svg" alt="Counter image"></li>
+                    <li>2015</li>
+                </ul>
+                <p class="product-details">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
```
Comment (Efiamotu-1) [inline index.html:61]: - [ ] please ensure you use the appropriate `color` which is `#344563` to style this highlighted code. It is important to stick to the figma design in relation to color and fonts as this is good practice and shows how much attention to details you pay as a skilled frontend developer :+1: :sparkles: 
- [ ] Please kindly make this change in every card for the work section so your overall design matches the figma design completely.

File: index.html
Comment (Efiamotu-1) [inline index.html:1]: - [ ] please kindly implement the button interaction as shown in the image below for when the buttons on your portfolio page is hovered on or clicked. It is important to add interactivity to your web page and this is one way to do so with the buttons
- figma button interaction
btn
- [ ] You can make use of the provided css `pseudo classes` such as `:active, :hover` to achieve this functionality :+1: :sparkles:

File: index.html [hunk 4]
```diff
@@ -40,5 +44,191 @@ <h4>LET’S CONNECT</h4>
...
+                        <li class="languages-items-li">
+                            <div class="languages-items">
+                                <img src="assets/images/html.png" alt="">
+                                <p>HTML</p>
+                            </div>
+                        </li>
+                        <li class="languages-items-li">
+                            <div class="languages-items">
+                                <img src="assets/images/css.png" alt="">
+                                <p>CSS</p>
```
Comment (Efiamotu-1) [inline index.html:191]: - [ ] I kindly suggest you make use of the `ul` and `li` tag to display the technologies of each categories such as `html, css javascript`. :+1:

File: index.html [hunk 5]
```diff
@@ -40,5 +44,191 @@ <h4>LET’S CONNECT</h4>
...
+
+
+    </section>
+    <section class="about-my-self main-section" id="about-my-self main-section">
+        <div class="div-description-about">
+            <h2 class="about-header">About myself</h2>
+            <p class="about-text">Hello I’m a software developer! I can help you build a product , feature or website.
+                Look through some of my work and
+                experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
```
Comment (Efiamotu-1) [inline index.html:153]: - [ ] please make use of the appropriate colors as shown in the figma image below
- figma design
fo
- your design
fc

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @kambereBr , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 5 above)
Comment (kambereBr) [reply index.html:153]: Thanks a lot @Efiamotu-1

Comment (shahadat3669) [inline index.html:153]: Well done on implementing the buttons and meeting the other requirements! 👍 However, there are a couple of additional changes that need to be made based on the previous reviewer's request:

- [ ] The "LET'S CONNECT" button should have the color `#7F8CFF` as per the design. Please update the button's color accordingly.
- [ ] Additionally, the paragraph in the about section should have the color `#344563` as requested by the previous reviewer. Please ensure that the paragraph's color matches the design.

By addressing these changes, you will bring the elements in line with the design specifications. Keep up the great work! 🎉

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @kambereBr , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights

✔️ No linter Errors
✔️ Descriptive README
✔️ Descriptive PR title and description
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 5 above)
Comment (kambereBr) [reply index.html:153]: Thanks a lot dear @shahadat3669

**Review by Cabraham1 — APPROVED** [review]
Hi @kambereBr  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kambereBr** [comment]
Thanks dear reviewers @shahadat3669 , @Cabraham1 , @Efiamotu-1

**Pull request merged**